prometheus/rootfs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM prom/prometheus:0.16.1
+FROM prom/prometheus:master
```

Comment:
Is there a reason to do this? It would be better to pin this so builds a reproducible